api/migrations/20210812174148-create-guild.js

```diff
@@ -12,27 +12,31 @@ module.exports = {
         defaultValue: ".",
         type: Sequelize.STRING,
       },
+      defaultMemberRoles: {
+        defaultValue: [],
+        type: Sequelize.ARRAY(Sequelize.STRING),
+      },
       opRoles: {
         defaultValue: [],
         type: Sequelize.ARRAY(Sequelize.STRING),
       },
       levels: {
         defaultValue: "[]",
-        type: Sequelize.STRING,
+        type: Sequelize.STRING(1024),
       },
       invites: {
         defaultValue: "[]",
-        type: Sequelize.STRING,
+        type: Sequelize.STRING(1024),
       },
       threadInactivityTime: {
-        defaultValue: 1000 * 60 * 60 * 24 * 7, //milliseconds
+        defaultValue: 60 * 24 * 2, // 2 days in minutes
```

Comment:
I thought 24 means 2 days?

Reply:
I am not sure what your question is asking.
60 minutes * 24 = 24 hours * 2 = 2 days.

Reply:
Never mind you are right.